recipes/minizip/all/patches/0014-Use-libbsd-overlay-component.patch

```diff
@@ -0,0 +1,25 @@
+From 31f86202ac1f9a9fad5c9b275289a5e608d281ed Mon Sep 17 00:00:00 2001
+From: ItsBasi <5033630+ItsBasi@users.noreply.github.com>
+Date: Fri, 14 Aug 2020 21:27:37 +0200
+Subject: [PATCH] Use libbsd-overlay component
+
+---
+ CMakeLists.txt | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 0212a88..61da18e 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -248,7 +248,7 @@ if(UNIX)
+         endif()
+ 
+         if(MZ_LIBBSD AND NOT HAVE_ARC4RANDOM_BUF)
+-            find_package(libbsd REQUIRED)
++            find_package(libbsd REQUIRED COMPONENTS libbsd-overlay)
```

Comment:
I don't know, but don't you also need to also link to `libbsd::libbsd-overlay` target later on?

Reply:
I guess i need to do it for the libbsd_COMPILE_DEFINITIONS. I should probably replace all xxx_INCLUDE_DIRS/xxx_LIB_DIRS and manual library naming with the conan generated targets like BZip2::BZip2